### ISSUE TITLE
Is istance vs. instance an intentional choice?

### ISSUE THREAD

**Issue by cclauss** [ms-jpq/coq_nvim#59]
codespell - -ignore-words-list="thats,wit" --skip="\*.po,\*.yml"
```
./coq_nvim/coq/server/reviewer.py:55: istance ==> instance
./coq_nvim/coq/server/trans.py:113: istance ==> instance
./coq_nvim/coq/shared/runtime.py:48: istance ==> instance
./coq_nvim/coq/shared/types.py:31: Trigged ==> Triggered
./coq_nvim/coq/shared/types.py:33: Trigged ==> Triggered
```

**Comment by jdhao** [comment]
I do not think so. They are typos.

**Comment by ms-jpq** [comment]
no im a dumb ass who cant spell. ill fix them, tank you for noticing

**Comment by UltiRequiem** [comment]
Going go be fixed when #68 get merged
